x-pack/plugins/cases/public/components/markdown_editor/use_plugins.ts

```diff
@@ -40,9 +40,9 @@ export const usePlugins = (disabledPlugins?: string[]) => {
     if (
       kibanaConfig?.markdownPlugins?.lens &&
       !disabledPlugins?.includes(LensPluginId) &&
-      appCapabilities?.visualize &&
+      appCapabilities?.visualize.crud &&
       // TODO remove this check after the lens plugin fixes this bug
-      appCapabilities.dashboard
+      appCapabilities?.dashboard.crud
```

Comment:
After this https://github.com/elastic/kibana/pull/127963 merged we can remove this additional dashboard check.